lightyear/src/transport/udp.rs

```diff
@@ -98,11 +98,19 @@ pub struct UdpSocketBuffer {
 
 impl PacketSender for UdpSocketBuffer {
     fn send(&mut self, payload: &[u8], address: &SocketAddr) -> Result<()> {
-        self.socket
-            .as_ref()
-            .lock()
-            .unwrap()
-            .send_to(payload, address)?;
+        let socket = self.socket.as_ref().lock().unwrap();
+
+        #[cfg(target_os = "windows")]
+        {
+            let mut peek_buf = [0u8; 1];
+            match socket.peek_from(&mut peek_buf) {
```

Comment:
That's great, could you please add a comment explaining your findings (e.g. that on windows the error happens in send and not receive, etc.) and what this code is suppsoed to fix? (the logspam if the remote udp receiver closes the socket)